src/main/webapp/help.html

```diff
@@ -6,17 +6,18 @@
     <link rel="stylesheet" href="assets/css/custom_style.css">
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
     <title>Help for Sections Manager</title>
+    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
```

Comment:
A more recent version of jQuery can be used.  For example:

https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js

Or maybe even newer than that.  See what's available.  If this is only used on this page, get the most recent version that is compatible with the other JS used on this page.